### DISCUSSION TITLE
Unable to pass student verification, even adter uploading correct information.

### DISCUSSION THREAD

**Discussion by Ve-Dev** [community/community#52261] [GitHub Education]
![Screenshot 2023-04-07 192703](https://user-images.githubusercontent.com/124495895/230621659-b704e436-8f5c-47f5-ad70-10d6719b957e.png)

I have tried applying for student developer pack with my email since my institute does not issue emails. I tried uploading my student ID which was rejected, even though it has the year of admission. 

I have read thorugh several similar discussions, and have tried the following :

- Taking a photo of the ID through webcam
- Using mobile to take image
- Tried additional document (Receipt from bursar) which was also rejected.

Note: The information provided was in English.

I have also tried to contact github support, but unable to send a support ticket since they have disabled tickets for manual review of applications. How can I get my application accepted if I'm unable to pass the AI even after submitted clearly legible documents, and having no way to contact support?

**Comment by samus-aran** [comment]
Hi @Ve-Dev 

Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

**Reply by Ve-Dev** [reply]
Ok, appreciate the response. Hope the issue gets resolved.

**Reply by Ve-Dev** [reply]
> Hi @Ve-Dev
> 
> Thank you for bringing this to our attention. We've passed this along to our internal team to investigate and when we have any updates we'll update this thread. 👍

Any updates? It's been a month now.
